src/domains/job/search/components/WriteJobSearchPostingForm.jsx

```
@@ -8,24 +8,22 @@ import EmploymentType from '../../common/components/form/EmploymentType'
 import JobCategory from '../../common/components/form/JobCategory'
 import WorkExperience from './form/WorkExperience'
 import ContactEmail from './form/ContactEmail'
-
-const WriteJobSearchPostingForm = () => {
+import useAuthStore from '../../../../store/login/useAuthStore'
+const WriteJobSearchPostingForm = ({ defaultValues, onSubmit }) => {
```

ESLint

[error] 'defaultValues' is missing in props validation

(react/prop-types)